rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -512,6 +632,30 @@ private void mockCompleteTxWithThrowForEmptying(Wallet wallet, Address expectedA
         }).when(wallet).completeTx(any(SendRequest.class));
     }
 
+    private void mockCompleteTxWithThrowForBuildBatchedPegout(Wallet wallet, List<ReleaseRequestQueue.Entry> entries, Throwable t) throws InsufficientMoneyException {
```

Comment:
Generic question here... do we have a test using the actual Wallet class? If we don't do it there may be conditions we are not properly testing.
Setting up the Wallet to generate conditions is not too hard. Please consider if we could do that instead of mocking, when the balance makes sense.

Reply:
Yes We Do Have a Test With The Actual Wallet Class, However All Exceptions Are Caught In ReleaseTransactionBuilder Class, Hence Reason Why I'm Testing It This Way